src/UI/VersionView.re

```diff
@@ -0,0 +1,154 @@
+/*
+ * VersionView.re
+ *
+ * Component for the 'version' buffer renderer
+ */
+
+open Revery.UI;
+open Oni_Model;
+open Oni_Core;
+
+module Model = Oni_Model;
+
+module Styles = {
+  let container = (~theme: Theme.t) =>
+    Style.[
+      backgroundColor(theme.editorBackground),
+      color(theme.foreground),
+      flexDirection(`Column),
+      flexGrow(1),
+      justifyContent(`Center),
+      alignItems(`Center),
+      overflow(`Hidden),
+    ];
+
+  let versionText = (~theme: Theme.t, ~font: UiFont.t) =>
+    Style.[
+      fontFamily(font.fontFile),
+      fontSize(12.),
+      color(theme.foreground),
+      marginTop(0),
+    ];
+};
+
+module HeaderView = {
+  module Styles = {
+    let header = (~uiFont: UiFont.t) =>
+      Style.[
+        flexGrow(0),
+        flexDirection(`Column),
+        justifyContent(`Center),
+        alignItems(`Center),
+        marginTop(16),
+        marginBottom(0),
+        fontFamily(uiFont.fontFile),
+        fontSize(14.),
+      ];
+  };
+
+  let make = (~text, ~state: State.t, ()) => {
+    <Text style={Styles.header(~uiFont=state.uiFont)} text />;
+  };
+};
+
+module VersionView = {
+  module Styles = {
+    open Style;
+
+    let container = [
+      flexDirection(`Row),
+      justifyContent(`Center),
+      alignItems(`Center),
+      height(18),
+      maxWidth(400),
+      minWidth(150),
+      flexGrow(0),
+      borderBottom(~color=Revery.Color.rgba(1.0, 1.0, 1.0, 0.2), ~width=1),
+    ];
+
+    let versionText = (~theme: Theme.t, ~fontFile, ~fontSize) => [
+      fontFamily(fontFile),
+      Style.fontSize(fontSize),
+      color(theme.foreground),
+    ];
+
+    let spacer = Style.[flexGrow(1)];
+    let tempSpacer = Style.[flexGrow(1)];
```

Comment:
Why are these distinguished? What's temporary about the `tempSpacer`?

Reply:
Oops, good question... I was using this to hack while tweaking the styles. Removed and factored out some inline styles 👍